server/plugin/webhook.go

```diff
@@ -1382,3 +1388,47 @@ func (p *Plugin) postWorkflowJobEvent(event *github.WorkflowJobEvent) {
 		}
 	}
 }
+
+func (p *Plugin) makeBotPost(message, postType string) *model.Post {
+	return &model.Post{
+		UserId:  p.BotUserID,
+		Type:    postType,
+		Message: message,
+	}
+}
+
+func (p *Plugin) postReleaseEvent(event *github.ReleaseEvent) {
+	if event.GetAction() != actionCreated && event.GetAction() != actionDeleted {
+		return
+	}
+
+	repo := event.GetRepo()
+	subs := p.GetSubscribedChannelsForRepository(repo)
+
+	if len(subs) == 0 {
+		return
+	}
+
+	newReleaseMessage, err := renderTemplate("newReleaseEvent", event)
```

Comment:
@phoinixgrr FYI this feature in the GitHub plugin will allow us to create subscriptions to new releases